GoogleSignIn/Tests/Unit/GIDTokenTest.m

```diff
@@ -36,6 +36,16 @@ - (void)testInitializer {
   XCTAssertEqualObjects(token.tokenString, tokenString);
   XCTAssertEqualObjects(token.expirationDate, _date);
 }
+
```

Comment:
You should add a test documenting how two tokens are equal if their token strings are equal but don't have expirations. You should also add a test showing that two tokens with equal strings, but one with an expiration and another without an expiration, are not equal.